### DISCUSSION TITLE
MA5 expert mode v.s. normal mode for plots

### DISCUSSION THREAD

**Discussion by shangliang** [MadAnalysis/madanalysis5#220] [General]
Hi, 
I am wondering if MA5 expert mode can generate distribution plots for both signal and background just as MA5 normal mode does?
Such as in the my_samples.list for MA5 expert input file, we define these lines,

import ../MG5_aMC_v3_4_2/proc_mssm_signal/Events/ca_r1d5_jptm200/tag_1_delphes_events.lhco.gz as s1
import ../MG5_aMC_v3_4_2/proc_sm_hjets/Events/run_01/tag_1_delphes_events.lhco.gz as b1
import ../MG5_aMC_v3_4_2/proc_sm_sigtop/Events/run_01/tag_1_delphes_events.lhco.gz as b2

here s1 is signal, and b1 and b2 are backgrounds. So we can get, such as, PT ETA, etc, distribution at the same plot.

**Comment by jackaraz** [comment]
Hi @shangliang, expert mode only produces the necessary files that include the information required for the histogram but does not produce the plot for you. You can read and plot expert mode outputs using the [``ma5-expert``](https://github.com/MadAnalysis/ma5_expert) package.

**Reply by shangliang** [reply]
Hi, Jack. Thank you for your message. I will plot these distributions with the help of ma5-expert package. Best, Liang.
